<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NCStrategyMainWin</class>
 <widget class="QMainWindow" name="NCStrategyMainWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1419</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NCStrategyMainWin</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextBrowser" name="MessagesBrowser">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>360</y>
      <width>871</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="SensersGroupBox">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>30</y>
      <width>871</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string>Sensors</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>252</width>
       <height>27</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="ImageTESTLayout">
      <item>
       <widget class="QLabel" name="ImageTESTLabel">
        <property name="text">
         <string>Image_TEST Num:</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="ImageTESTLineEdit"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>252</width>
       <height>27</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="SendTESTLayout">
      <item>
       <widget class="QLabel" name="SendTESTLabel">
        <property name="text">
         <string>Send_TEST Num:</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="SendTESTLineEdit"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="MissionGroupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>461</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Mission</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>252</width>
       <height>27</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="CurMissionLayout">
      <item>
       <widget class="QLabel" name="CurMissionLabel">
        <property name="text">
         <string>Current Mission:</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="CurMissionLineEdit"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>251</width>
       <height>27</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="CurStageLayout">
      <item>
       <widget class="QLabel" name="CurStageLabel">
        <property name="text">
         <string>Current Stage:</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="CurStageLineEdit"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>120</y>
       <width>421</width>
       <height>27</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="StageMessageLabel">
        <property name="text">
         <string>Stage Message:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="StageMessageLineEdit"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="ServerGroupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>461</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>Server INPUT</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>381</width>
       <height>29</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="ServerCMDLayout">
      <item>
       <layout class="QHBoxLayout" name="ServerCMDLayout_2">
        <item>
         <widget class="QLabel" name="ServerCMDLabel">
          <property name="text">
           <string>Server CMD:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ServerCMDLineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="ServerCMDSendButton">
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="ServerRunButton">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>220</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Server Run</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1419</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
